### PR TITLE
Helm 3 support.

### PR DESCRIPTION
**Pull request by gmorse81**
Also removes git cloning support since it never really worked right anyways.

### CODE REVIEW

File: boondoggle/reqbuild.go [hunk 1]
```diff
@@ -53,6 +87,50 @@ func BuildRequirements(b Boondoggle, svo []string) Requirements {
 	return r
 }
 
+// WriteRequirements writes the dependencies to eithe chart.yaml or requirements.yaml depending on the version of helm you are running
```
Comment (johnhoodjr) [inline boondoggle/reqbuild.go:90]:
```suggestion
// WriteRequirements writes the dependencies to either chart.yaml or requirements.yaml depending on the version of helm you are running
```

File: cmd/up.go [hunk 2]
```diff
@@ -33,13 +30,11 @@ var upCmd = &cobra.Command{
...
 
-		//Build requirements.yml
+		// Build Requirements struct
 		r := boondoggle.BuildRequirements(b, viper.GetStringSlice("state-v-override"))
 
-		// Write the new requirements.yml
-		out, err := yaml.Marshal(r)
-		path := fmt.Sprintf("%s/requirements.yaml", b.Umbrella.Path)
-		ioutil.WriteFile(path, out, 0644)
+		// Write the new requirements.yml or chart.yaml
```
Comment (johnhoodjr) [inline cmd/up.go:36]:
```suggestion
		// Write the new requirements.yaml or chart.yaml
```

**Review by johnhoodjr — CHANGES_REQUESTED** [review]

**Review by johnhoodjr — APPROVED** [review]

File: boondoggle/helm.go [hunk 3]
```diff
@@ -65,18 +65,22 @@ func (b *Boondoggle) DoUpgrade(namespace string, release string, dryRun bool, us
 	}
 
 	// Add a longer timeout
-	chunk := "--timeout 1800 --wait"
+	chunk := "--timeout 1800s --wait"
```
Comment (codycraven) [inline boondoggle/helm.go]: Verify helm 2 works with quantifier.

File: boondoggle/helm.go [hunk 4]
```diff
@@ -201,3 +205,11 @@ func (b *Boondoggle) SelfFetch(path string, version string) error {
 	}
 	return nil
 }
+
+func (b Boondoggle) is2() bool {
+	fmt.Println(b.HelmVersion)
```
Comment (johnhoodjr) [inline boondoggle/helm.go]:
```suggestion
```

**Review by johnhoodjr — CHANGES_REQUESTED** [review]

**Pull request merged**